buddyboss/buddyboss-platform
# PROD-650 #3265-fix-Extra character when the discussion subscribe button was toggle.

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3265 

### How to test the changes in this Pull Request:

1. Go to any discussion
2. Change the desktop to tablet/mobile view.
3. Click on the subscribe/unsubscribe button.
4. See error extra "|" character added.

### Proof Screenshots or Video

https://user-images.githubusercontent.com/83228316/147747261-a3bcf847-4464-4aa9-a1e6-0a51d587a282.mp4

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
